Sources/MeiliSearch/Documents.swift

```diff
@@ -16,18 +16,24 @@ struct Documents {
   func get<T>(
     _ uid: String,
     _ identifier: String,
+    fields: [String]? = nil,
     _ completion: @escaping (Result<T, Swift.Error>) -> Void)
   where T: Codable, T: Equatable {
-    let query: String = "/indexes/\(uid)/documents/\(identifier)"
+    var query: String = "/indexes/\(uid)/documents/\(identifier)"
+
+    if fields != nil {
+      query.append(fields?.joined(separator: ",") ?? "")
```

Comment:
You are correct, I was concerned about how Swift will parse JSON objects into structs that miss some fields. And my concern was confirmed because Swift breaks when the struct has missing fields in non-optional properties.

And then I totally miss to add this assertion: `XCTAssertEqual(self.session.nextDataTask.request?.url?.query, "fields=title,description")`